INSTALL.md

```diff
@@ -140,19 +140,51 @@ pecl install oci8-2.0.10
 ```
 This will download and compile the module, and place it in your php extension dir.
 
-Add the ```extension=oci8.so``` line to your php.ini or create a configuration file: 
+Add the ```extension=oci8.so``` line to your php.ini or create a configuration file:
 
-```bash 
+```bash
 echo 'extension=oci8.so' > /etc/php.d/oci8.ini
 ```
 
 Confirm it is working with ```php -i | grep -i oci8```
 
+#### MariaDB/mysql
+The following instructions are to set up a local MYSQL database for GOCDB. They have not been tested in a production enviroment and are currently intended for test instances.
+
+First you will need to install MariaDB. Then start the 'mysqld' service.
+
+Access the CLI for MariaDB
+
+```
+$ mysql
+```
+
+Create a database to use
+
+````
+CREATE DATABASE gocdb DEFAULT CHRACTER SET utf8 DEFAULT COLLATE utf8;
+````
+
+create a user to access it and give that user a password
+
+````
+CREATE USER 'gocdbuser'@'localhost' IDENTIFIED BY 'PASSWORD_GOES_HERE';
+````
+
+Grant the user permissions over the database. Note: these permissions are wider than is strictly required and deployment into a production environment would first require revising these permissions more in line with those given for Oracle in the set up instructions.
+
+````
+GRANT ALL PRIVILEGES ON gocdb.* TO 'gocdbuser'@'localhost';
+FLUSH PRIVILEGES
```

Comment:
`FLUSH PRIVILEGES` is only necessary if you update privileges by directly manipulating the grant tables. Indirect methods like `GRANT` will handle that for you.

Reply:
Did that change at some point? Because I'm sure it used to be required.

Reply:
This still seems to have been the case in 5.1 and even 5.0. `FLUSH PRIVILEGES` is only required if you do a direct manipulation of the `user` table:

> [With `CREATE USER`, `FLUSH PRIVILEGES` is unnecessary.](http://download.nust.na/pub6/mysql/doc/refman/5.0/en/adding-users.html)

So now we're talking around at least 13 years. It may just be the case that it's commonly mistaken.